### ISSUE TITLE
Remove temporary file CircularBufferUint2.sol from proof.sol.

### ISSUE THREAD

**Issue by deepesh-kn** [OpenST/compute#99]
The changes related to `CircularBufferUint` is being done in https://github.com/mosaicdao/mosaic-1/pull/74
This touches lot of contracts.
For this ticket https://github.com/mosaicdao/mosaic-1/pull/97 `CircularBufferUint` was needed. 
So a temporary contract `CircularBufferUint2` was used in Proof.sol (Message bus), to avoid overlapping changes and avoid rebasing.

Once https://github.com/mosaicdao/mosaic-1/pull/74 is merged, then delete `CircularBufferUint2` and use `CircularBufferUint`.

**Comment by deepesh-kn** [comment]
This is not needed anymore. The dependent changes are already done.
